packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -295,6 +309,14 @@ export default defineComponent({
       this.filesInEstimation[file.meta.uploadId] = progress.bytesUploaded
 
       const timeElapsed = +new Date() - this.timeStarted
+
+      // "bytesTotal" is a wrong type documentation,
+      // fix in progress via https://github.com/transloadit/uppy/pull/4263
+      this.uploadSpeed = getSpeed({
```

Comment:
I had the content of `getSpeed` copied to a local method while figuring out why it wouldn't do what I expected it to, using it directly still returns `NaN` and breaks the tooltip so further investigation needed/potentially a type error somewhere

Reply:
upstream fix has been released.

Reply:
Ah nice, thanks for the reminder! I'll see if I can get this PR done this week then :)